libs/ui/src/examples/Fetch/E3_FetchRequest.tsx

```diff
@@ -3,14 +3,26 @@ import { useEffect, useState } from 'react'
 
 import { Box, Stack } from '@chakra-ui/react'
 import { API_URL, TOKEN } from '@wsvvrijheid/config'
+import { StrapiModel } from '@wsvvrijheid/types'
 import { request } from '@wsvvrijheid/utils'
 
 export const FetchRequest = () => {
   const [blogs, setBlogs] = useState([])
 
   useEffect(() => {
+    ;(async () => {
+      const response = await request({ url: 'api/blogs', token: TOKEN })
```

Comment:
I always recommend specifying type of the data.

```suggestion
      const response = await request<Blog[]>({ url: 'api/blogs', token: TOKEN })
```